### PR TITLE
[DXP-1186] Adjust published messages to new messages sending way

### PR DESCRIPTION
**Pull request by andrzejmikolajczakds**
Fetch schema before each publish message.

This PR has to be check by e2e tests based on docker images when their will be released

### CODE REVIEW

File: core/src/test/java/dev/streamx/cli/ingestion/publish/PublishPayloadCommandTest.java [hunk 1]
```diff
@@ -231,17 +238,40 @@ public void shouldPublishReplacedJsonPathWithBinaryValue(QuarkusMainLauncher lau
 
     // then
     expectSuccess(result);
+    StringValuePattern matchingPngFileContent = matchingJsonPath(
+        "$.payload[\"dev.streamx.blueprints.data.Page\"].content.bytes",
```
Comment (jgardo-ds) [inline core/src/test/java/dev/streamx/cli/ingestion/publish/PublishPayloadCommandTest.java:242]: FYI: przedrostek `$.` jest opcjonalny - bez tego też powinno śmigać

File: pom.xml [hunk 2]
```diff
@@ -39,7 +39,8 @@
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
     <quarkus.platform.version>3.13.1</quarkus.platform.version>
 
-    <streamx.version>0.0.33</streamx.version>
+    <!--  TODO change it to released version when ready  -->
+    <streamx.version>0.0.35-SNAPSHOT</streamx.version>
```
Comment (jgardo-ds) [inline pom.xml:43]: It has just been released! 🚀

File: core/src/main/java/dev/streamx/cli/ingestion/publish/IngestionMessageJsonFactory.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  ) {
+
+    ObjectMapper mapper = new ObjectMapper();
+    ObjectNode root = mapper.createObjectNode();
+
+    root.put("key", key);
+    root.put("action", action);
+    root.putNull("eventTime");
+    root.putObject("properties");
+    ObjectNode properties = root.putObject("payload");
```
[the remark this answers is not in the document]
Comment (andrzejmikolajczakds) [reply core/src/main/java/dev/streamx/cli/ingestion/publish/IngestionMessageJsonFactory.java]: done

File: core/src/test/java/dev/streamx/cli/ingestion/publish/PublishPayloadCommandTest.java (hunk 1 above)
Comment (andrzejmikolajczakds) [reply core/src/test/java/dev/streamx/cli/ingestion/publish/PublishPayloadCommandTest.java]: done

File: core/src/main/java/dev/streamx/cli/ingestion/publish/PublishCommand.java [hunk 4]
```diff
@@ -34,6 +42,15 @@ public class PublishCommand extends BaseIngestionCommand {
   @Inject
   PayloadResolver payloadResolver;
 
+  @Inject
+  SchemaProvider schemaProvider;
+
+  @Spec
```
[the remark this answers is not in the document]
Comment (andrzejmikolajczakds) [reply core/src/main/java/dev/streamx/cli/ingestion/publish/PublishCommand.java]: thx

File: pom.xml (hunk 2 above)
Comment (andrzejmikolajczakds) [reply pom.xml]: great, changed in new PR

File: core/src/main/java/dev/streamx/cli/ingestion/BaseIngestionCommand.java [hunk 5]
```diff
@@ -37,9 +37,9 @@ public abstract class BaseIngestionCommand implements Runnable {
   @Override
   public final void run() {
     try (StreamxClient client = streamxClientProvider.createStreamxClient(ingestionClientConfig)) {
-      Publisher<JsonNode> publisher = client.newPublisher(getChannel(), JsonNode.class);
-      perform(publisher);
-    } catch (StreamxClientUnsupportedChannelException e) {
+      Publisher<JsonNode> ingestor = client.newPublisher(getChannel(), JsonNode.class);
```
Comment (jgardo-ds) [inline core/src/main/java/dev/streamx/cli/ingestion/BaseIngestionCommand.java:40]: rename intestor -> publisher

File: core/src/main/java/dev/streamx/cli/ingestion/publish/PublishCommand.java [hunk 6]
```diff
@@ -42,12 +56,44 @@ protected String getChannel() {
   @Override
   protected void perform(Publisher<JsonNode> publisher) throws StreamxClientException {
     List<PayloadArgument> mergedPayloadArgumentList = prependPayloadFile();
-    JsonNode jsonNode = payloadResolver.createPayload(mergedPayloadArgumentList);
-    PublisherSuccessResult result = publisher.publish(publishTargetArguments.getKey(), jsonNode);
+    JsonNode payload = payloadResolver.createPayload(mergedPayloadArgumentList);
+    JsonNode schemaJson = null;
```
Comment (jgardo-ds) [inline core/src/main/java/dev/streamx/cli/ingestion/publish/PublishCommand.java:60]: dej to do metody wyliczanie tego `schemaJson`

Comment (andrzejmikolajczakds) [reply core/src/main/java/dev/streamx/cli/ingestion/publish/PublishCommand.java]: done

**Review by jgardo-ds — APPROVED** [review]

**Pull request merged**